Sources/App/configure.swift

```diff
@@ -7,13 +7,28 @@ public func configure(_ app: Application) throws {
     // uncomment to serve files from /Public folder
     // app.middleware.use(FileMiddleware(publicDirectory: app.directory.publicDirectory)){{#fluent}}
 
+    {{#fluent.db.is_postgres}}
+    let port: Int
```

Comment:
this gets defined under postgres configuration but used under both postgres and mysql configuration below so that's going to be a problem. Maybe this code should live under all database configurations (there's a way in mustache to ask whether the template is using any database at all, but I forget what it is off the top of my head) and it should set an optional `port` property that can be nil coalesced below to the appropriate port for each given database (`5432` for postgres and `3306` for mysql).

Reply:
Found it; the logic used by the `docker-compose.yml` file is "configured for Fluent but not configured for SQLite."

```
{{#fluent}}{{^db.is_sqlite}}
  ...
{{/db.is_sqlite}}{{/fluent}}
```

Reply:
I went with Gwynne's suggestion.